File: Anki2/Anki2/Models/Usuario.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace Anki2.Models
{
    public class Usuario
    {
        private int codUsuario;
        private string nomeUsuario;
        private string email;
        private string senha;

        public int CodUsuario
        {
            get
            {
                return codUsuario;
            }

            set
            {
                codUsuario = value;
            }
        }

        public string NomeUsuario
        {
            get
            {
                return nomeUsuario;
            }

            set
            {
                nomeUsuario = value;
            }
        }

        public string Email
        {
            get
            {
                return email;
            }

            set
            {
                email = value;
            }
        }

        public string Senha
        {
            get
            {
                return senha;
            }

            set
            {
                senha = value;
            }
        }

        public Usuario() { }

        public Usuario(int codUsuario, string nomeUsuario, string email, string senha)
        {
            this.codUsuario = codUsuario;
            this.nomeUsuario = nomeUsuario;
            this.email = email;
            this.senha = senha;
        }
    }
}

File: AnkiOABJava/app/src/main/java/lucas/com/br/ankioab/Usuario.java
package lucas.com.br.ankioab;

/**
 * Created by aluno on 07/06/2017.
 */

public class Usuario {

    private Integer CodUsuario ;
    private String NomeUsuario ;
    private String Email ;
    private String Senha ;

    public Integer getCodUsuario() {return CodUsuario;}

    public void setCodUsuario(Integer codUsuario) {this.CodUsuario = codUsuario;}

    public String getNomeUsuario() {return NomeUsuario;}

    public void setNomeUsuario(String nomeUsuario) {this.NomeUsuario = nomeUsuario;}

    public String getEmail() {return Email;}

    public void setEmail(String email) {this.Email = email;}

    public String getSenha() {return Senha;}

    public void setSenha(String senha) {this.Senha = senha;}

}
